### DISCUSSION TITLE
action suggestion: shell command

### DISCUSSION THREAD

**Discussion by GlassBricks** [Xoriun/Notes-Creator#2] [Ideas]
A simple but flexible action could be to run some shell command. This could provide an entry point into basically everything else (screenshot, authotkey, etc.)

**Comment by GlassBricks** [comment]
This might have some security risk if the scripts are shared, though.

**Comment by Xoriun** [comment]
Nice idea, I will look into it. However, I am currently working on improving the edit window.
EDIT: moved to ideas
